cou/utils/nova_compute.py

```diff
@@ -85,3 +99,15 @@ async def verify_empty_hypervisor(unit: Unit, model: Model) -> None:
             unit_instance_count,
         )
         raise HaltUpgradeExecution(f"Unit: {unit.name} has {unit_instance_count} VMs running")
+
+
+def stringify_units(units: Iterable[Unit]) -> str:
```

Comment:
non-blocking: If we are only going to use this function here, let's mark it as a private function
```suggestion
def _get_stringify_units(units: Iterable[Unit]) -> str:
```

Also do we really need the argument type to be `Iterable` instead of `set`?